diff --git a/ssl/ssl_cipher.cc b/ssl/ssl_cipher.cc
index a420f4d..00c10c4 100644
--- a/ssl/ssl_cipher.cc
+++ b/ssl/ssl_cipher.cc
@@ -263,18 +263,6 @@ static constexpr SSL_CIPHER kCiphers[] = {
 
     // TLS 1.3 suites.
 
-    // Cipher 1301
-    {
-      TLS1_TXT_AES_128_GCM_SHA256,
-      "TLS_AES_128_GCM_SHA256",
-      TLS1_CK_AES_128_GCM_SHA256,
-      SSL_kGENERIC,
-      SSL_aGENERIC,
-      SSL_AES128GCM,
-      SSL_AEAD,
-      SSL_HANDSHAKE_MAC_SHA256,
-    },
-
     // Cipher 1302
     {
       TLS1_TXT_AES_256_GCM_SHA384,
diff --git a/ssl/tls13_server.cc b/ssl/tls13_server.cc
index 8fb0d5c..d7c9990 100644
--- a/ssl/tls13_server.cc
+++ b/ssl/tls13_server.cc
@@ -109,8 +109,7 @@ class CipherScorer {
  public:
   CipherScorer(uint16_t group_id)
       : aes_is_fine_(EVP_has_aes_hardware()),
-        security_128_is_fine_(group_id != SSL_CURVE_CECPQ2 &&
-                              group_id != SSL_CURVE_CECPQ2b) {}
+        security_128_is_fine_(false) {}
 
   typedef std::tuple<bool, bool, bool> Score;
 
