diff --git a/ssl/tls13_server.cc b/ssl/tls13_server.cc
index 8fb0d5c..d7c9990 100644
--- a/ssl/tls13_server.cc
+++ b/ssl/tls13_server.cc
@@ -109,8 +109,7 @@ class CipherScorer {
  public:
   CipherScorer(uint16_t group_id)
       : aes_is_fine_(EVP_has_aes_hardware()),
-        security_128_is_fine_(group_id != SSL_CURVE_CECPQ2 &&
-                              group_id != SSL_CURVE_CECPQ2b) {}
+        security_128_is_fine_(false) {}
 
   typedef std::tuple<bool, bool, bool> Score;
